### PR TITLE
v1.3.0

### PR DESCRIPTION
**Pull request by adrienthebo**
This pull request prepares the Forseti module for the v1.3.0 release. This includes updating the `forseti_version` variable default and adding a CHANGELOG.md entry for v1.3.0.

This pull request will be a draft until forseti-security v1.3.0 has been released.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ Extending the adopted spec, each change should have a link to its corresponding
 
 ## [Unreleased]
```
Comment (aaron-lane) [inline CHANGELOG.md]: The URL backing this header needs to be updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,12 @@ Extending the adopted spec, each change should have a link to its corresponding
...
 
+## [v1.3.0]
+1.3.0 is a backwards compatible feature release. This module release supports Forseti v2.13.0.
```
Comment (aaron-lane) [inline CHANGELOG.md]: The change to the default value of `var.forseti_version` should be logged under CHANGED.

Note for posterity: the format of this file should be changed to match Keep a Changelog.

File: CHANGELOG.md (hunk 1 above)
Comment (adrienthebo) [reply CHANGELOG.md]: Fixed in 110908e

File: CHANGELOG.md (hunk 2 above)
Comment (adrienthebo) [reply CHANGELOG.md]: Fixed in 110908e.

Following up on the format changes, what diff would you like to see?

Comment (aaron-lane) [reply CHANGELOG.md]: I think it's just the uppercase headers, which is admittedly ultra aesthetic. :blush:

**Pull request merged**